docs/README.md

```diff
@@ -41,16 +45,16 @@ Stores all of your store loyalty cards on your phone, removing the need to carry
 [<img src="https://github.com/TheLastProject/Catima/raw/master/fastlane/metadata/android/en-US/images/phoneScreenshots/screenshot-07.png" width=250>](https://github.com/TheLastProject/Catima/raw/master/fastlane/metadata/android/en-US/images/phoneScreenshots/screenshot-07.png)
 [<img src="https://github.com/TheLastProject/Catima/raw/master/fastlane/metadata/android/en-US/images/phoneScreenshots/screenshot-08.png" width=250>](https://github.com/TheLastProject/Catima/raw/master/fastlane/metadata/android/en-US/images/phoneScreenshots/screenshot-08.png)
 
-# Migrating from other apps
+# Moving data from other apps
 
 [See our migration guides](migrate).
```

Comment:
I actually deleted the migration guides recently because there are now in-app instructions on how to get the files you need. This section probably needs a rewrite, maybe just mention what we support importing from and possible gotchas?